packages/core/src/generatePolicy.js

```diff
@@ -162,7 +174,7 @@ function createModuleInspector(opts) {
       moduleRecord.ast ||
       parse(moduleRecord.content, {
         // esm support
-        sourceType: 'module',
+        sourceType: 'unambiguous',
```

Comment:
what does this mean? if it's not ambiguous, why doesn't it have a different value?

Reply:
it means, "babel, please determine which module type this is".  I am not sure why it's called "unambiguous", though; it's a confusing property.